transport_nantes/topicblog/templates/topicblog/content_mlp.html

```diff
@@ -79,8 +79,11 @@ <h1 class="hero-title">{{ page.header_title }}</h1>
                 </a>
             </div>
           {% endif %}
-          {% show_mailing_list  mailinglist=page.mailing_list.mailing_list_token title=page.subscription_form_title %}
-
+          {% if page.mailing_list %}
+               {% show_mailing_list  mailinglist=page.mailing_list.mailing_list_token title=page.subscription_form_title %}
+          {% else %}
+               <p>Aucune liste de diffusion n'est disponible pour l'instant.</p>
```

Comment:
I find this visually a bit odd, since the user has to _remember_ that what is missing is a button.  Also, the "for the moment" doesn't quite make sense: what is the source of the temporality?  You're assuming the user remembers what all these things are supposed to do.  But it's your job to guide the user so that s/he remembers as it happens.

Instead, perhaps make an inactive button with "no list" as the button text.